theano/gof/op.py

```diff
@@ -966,7 +966,7 @@ def make_thunk(self, node, storage_map, compute_map, no_recycling):
         if new_node is not None:
             node = new_node
 
-        if self._op_use_c_code:
```

Comment:
That member was set in Op. So we would need to change all subclass of Op. So I don't see how to fix it otherwise without too much code duplicate. I don't like the idea of implementing Op.\__setstate__. It would be easy to not call it from subclass that implement it.